src/zkevm_specs/evm/execution/extcodecopy.py

```diff
@@ -0,0 +1,56 @@
+from ..instruction import Instruction, Transition
+from ...util.param import EXTRA_GAS_COST_ACCOUNT_COLD_ACCESS
+from ...util import N_BYTES_MEMORY_ADDRESS, FQ
+from ..table import AccountFieldTag, CallContextFieldTag, CopyDataTypeTag
+
+
+def extcodecopy(instruction: Instruction):
+    opcode = instruction.opcode_lookup(True)
+
+    address, memory_offset_word, code_offset, size_word = (
+        instruction.rlc_to_fq(instruction.stack_pop(), 20),
```

Comment:
You will need to import `N_BYTES_ACCOUNT_ADDRESS` from `...util.param`:
```suggestion
        instruction.rlc_to_fq(instruction.stack_pop(), N_BYTES_ACCOUNT_ADDRESS),
```

Reply:
I though I already suggested that. 
Thanks for catching it again!

Reply:
Fixed in [03be899](https://github.com/privacy-scaling-explorations/zkevm-specs/pull/257/commits/03be8994d3e706bde93bd2c9d0d276c98a93e182) :)